adhocracy-plus/config/urls.py

```diff
@@ -89,7 +89,7 @@
     re_path(r'^api/', include(likes_router.urls)),
     re_path(r'^api/', include(router.urls)),
     re_path(r'^api/login', obtain_auth_token, name='api-login'),
-    re_path(r'^api/account/', AccountViewSet.as_view(), name='account'),
+    re_path(r'^api/account/', AccountViewSet.as_view()),
```

Comment:
Upsi, right! You could name it api-account, though?! If we want a name.

Reply:
Did that first, but then thought we dont use the name anyway, so removed it?

Reply:
Do you want it back? :)

Reply:
No, I don't care. Where do we normally use it, though? In the tests? But anyway, can add it when we need it!

Reply:
Ah yes, we might need it for the tests, right! Well, too late :stuck_out_tongue_closed_eyes: